ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -122,6 +158,42 @@ export default class ExperimentalTab extends PureComponent {
                   }}
                 />
               </div>
+              <div className="settings-page__content-item-col settings-page__content-item-col__security-toggle-option">
+                <div>
+                  <Box display={Display.Flex}>
+                    <Text
+                      variant={TextVariant.bodyMd}
+                      color={TextColor.textDefault}
+                    >
+                      {t('openSeaLabel')}
+                    </Text>
+                    <Tag marginLeft={2} label="Beta" />
+                  </Box>
+                  <Text
+                    variant={TextVariant.bodySm}
+                    as="h6"
+                    color={TextColor.textAlternative}
+                    marginTop={0}
+                    marginRight={1}
+                  >
+                    {t('openSeaMessage')}
+                  </Text>
+                </div>
+                <ToggleButton
+                  value={transactionSecurityCheckEnabled}
+                  onToggle={(value) => {
+                    this.context.trackEvent({
+                      category: MetaMetricsEventCategory.Settings,
+                      event: 'Enabled/Disable TransactionSecurityCheck',
+                      properties: {
+                        action: 'Enabled/Disable TransactionSecurityCheck',
+                        legacy_event: true,
```

Comment:
```suggestion
                        enabled: !value,
```
suggestion to remove `legacy_event` and add `enabled` prop

Reply:
This I have kept same as before, I will avoid changing it.

Reply:
seems odd but okay, will consider this not a blocker since logic is preserved. We may want to check with someone about the enabled prop